interpreter_book/src/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/wwgberlin/book-club/interpreter_book/src
+
+go 1.14
+
+require github.com/Asalle/monkey-interpreter v0.0.0-20200628192903-c0e047c0d195
```

Comment:
I am not sure if I understand why this is needed :thinking: would you mind explaining to me?

Reply:
I dm-ed you asking how do you prefer to get info: here or personally

Reply:
If we do it here then its open for all to see and learn but if its better in a direct conversation perhaps a summary can be put here? (I'm interested in the outcome :) )